packages/cli/compiler/darwin.ts

```diff
@@ -11,4 +14,6 @@ export function run(filename = 'configu') {
   );
   execSync('codesign --sign - configu');
```

Comment:
<img width="1038" alt="image" src="https://github.com/user-attachments/assets/a998c2e0-8d33-439c-b002-9aecbfaf5f04">

 I think this is not working